page.html

```diff
@@ -1,147 +1,147 @@
-      <iframe id="video" height="300" width="500"
-        src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0" frameborder="0"
-        allowfullscreen=""></iframe>
-    </section>
-
-
-    <section id="prices" class="prices">
-      <div class="card-group">
-        <div class="card" id="card">
-          <div class="card-body">
-            <h5 class="card-title">Tenor Trombone</h5>
-            <p class="card-text">$600</p>
-            <p class="card-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum laboriosam facere
-              atque maxime molestias nihil deserunt similique, libero perferendis numquam ipsam tenetur quos, error et
-              exercitationem recusandae, provident accusantium fuga?</p>
-            <button>Buy</button>
-
-          </div>
-        </div>
-        <div class="card" id="card">
-          <div class="card-body">
-            <h5 class="card-title">Bass Trombone</h5>
-            <p class="card-text">$900</p>
-            <p class="card-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum laboriosam facere
-              atque maxime molestias nihil deserunt similique, libero perferendis numquam ipsam tenetur quos, error et
-              exercitationem recusandae, provident accusantium fuga?</p>
-            <button>Buy</button>
-
-          </div>
-        </div>
-        <div class="card" id="card">
-          <div class="card-body">
-            <h5 class="card-title">Valve Trombone</h5>
-            <p class="card-text">$1200</p>
-            <p class="card-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum laboriosam facere
-              atque maxime molestias nihil deserunt similique, libero perferendis numquam ipsam tenetur quos, error et
-              exercitationem recusandae, provident accusantium fuga?</p>
-            <div class="contenedor-btn">
-              <button>Buy</button>
-
-            </div>
-
-          </div>
-        </div>
-      </div>
-
-    </section>
-  </div>
-
-
-
-
-
-
-  <!-- Boostrap -->
-  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"
-    integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4"
-    crossorigin="anonymous"></script>
-
-  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-  <script src="js/index.js"></script>
-</body>
-
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/6e8f4ae605.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header id="header">
```

Comment:
- [OPTIONAL] When making reviews, kindly consider the following factors:

### 1. Your comments should sound polite. 

#### Example of impolite comments:

> "Use semantic tags"

#### Example of polite comments:

> "I think it would be better to use semantic tags"

### 2. You should give a good reason why you are making the suggestion

#### Example of unexplained comment:

> Why don't you use an article tag here instead of a div?

#### Example of properly explained comment:

> Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

### 3. Your comment should be clear and easy to understand

#### Example of vague comment: 

> The README is not descriptive, why don't you make it better?

#### Example of clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

#### Hint: I would advise you to make your comments meet these criteria above. 👍 💻